Content.Client/Administration/UI/AdminAnnounceWindow.xaml.cs

```diff
@@ -35,7 +37,8 @@ private void AnnouncementOnOnTextChanged(GUIBoundKeyEventArgs args)
         private void AnnounceMethodOnOnItemSelected(OptionButton.ItemSelectedEventArgs args)
         {
             AnnounceMethod.SelectId(args.Id);
-            Announcer.Editable = ((AdminAnnounceType?)args.Button.SelectedMetadata ?? AdminAnnounceType.Station) == AdminAnnounceType.Station;
+            Announcer.Editable = ((AdminAnnounceType?)args.Button.SelectedMetadata ?? AdminAnnounceType.Station) == AdminAnnounceType.Station
+            || ((AdminAnnounceType?)args.Button.SelectedMetadata ?? AdminAnnounceType.Antag) == AdminAnnounceType.Antag; // Frontier
```

Comment:
Nitpick: second ?? here isn't strictly necessary.  First line will already coalesce a null value into Station - values in second comparison will never be null.  If you want to keep the comparison the same for simplicity of reading, also makes sense.
```suggestion
            || ((AdminAnnounceType?)args.Button.SelectedMetadata) == AdminAnnounceType.Antag; // Frontier
```

Reply:
Just wanted it to be clear